ios/fastlane/Fastfile

```diff
@@ -18,6 +18,50 @@ default_platform(:ios)
 platform :ios do
   desc "Push a new beta build to TestFlight"
 
+  lane :checkBuild do
```

Comment:
Do we need the signing credentials available? If yes, if we switch this to debug can we avoid it?